src/NLog.Targets.Syslog/MessageSend/MessageTransmitter.cs

```diff
@@ -38,7 +38,9 @@ public static MessageTransmitter FromConfig(MessageTransmitterConfig messageTran
         protected MessageTransmitter(string server, int port)
         {
             Server = server;
-            IpAddress = Dns.GetHostAddresses(server).FirstOrDefault()?.ToString();
+            var hostAddresses = Dns.GetHostAddressesAsync(server);
+            hostAddresses.Wait();
+            IpAddress = hostAddresses.Result.FirstOrDefault()?.ToString();
```

Comment:
Well all sync-methods in Net Core has been removed, so that is why the awkward way (Because the Syslog-code expects it to be sync)